docs/Getting-Started.md

```diff
@@ -159,7 +159,6 @@ async function routes (fastify, options) {
     try {
       return await collection.findOne({ id: request.params.id })
     } catch (err) {
-      reg.log.error(err)
       return new Error('Something went wrong')
     }
```

Comment:
Imho we should do something different in this route, change what is returned by `collection.findOne()`  for example. This try-catch is pointless.